uportal-war/src/main/java/org/jasig/portal/portlets/account/selfedit/AccountPasswordFormValidator.java

```diff
@@ -1,100 +0,0 @@
-/**
- * Licensed to Apereo under one or more contributor license
- * agreements. See the NOTICE file distributed with this work
- * for additional information regarding copyright ownership.
- * Apereo licenses this file to you under the Apache License,
- * Version 2.0 (the "License"); you may not use this file
- * except in compliance with the License.  You may obtain a
- * copy of the License at the following location:
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package org.jasig.portal.portlets.account.selfedit;
-
-import org.apache.commons.lang.StringUtils;
-import org.jasig.portal.persondir.ILocalAccountDao;
-import org.jasig.portal.persondir.ILocalAccountPerson;
-import org.jasig.portal.security.IPortalPasswordService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.binding.message.MessageBuilder;
-import org.springframework.binding.message.MessageContext;
-import org.springframework.stereotype.Component;
-
-/**
- * 
- * @author Jen Bourey, jbourey@unicon.net
- * @version $Revision$
- */
-@Component("accountPasswordFormValidator")
-public class AccountPasswordFormValidator {
```

Comment:
I have some anxieties about this one.  I dimly remember that Spring form binding & validation support may include some kind of magic for finding (and invoking) validators.  But there's one issue with that, at least:  the HTML form for updating a password doesn't even include (currently) a box for the old password.  Also, this class doesn't implement any Spring interfaces... how would the magic know which method to call?  Anything starting with "validate*"?